src/aiortc/contrib/getkeypoints.py

```diff
@@ -0,0 +1,84 @@
+"""
+The following module extracts 68 tuples of facial keypoints. 
+"""
+
+# Importing libraries
+import torch
+from torchvision import transforms
+from PIL import Image
+import pathlib
+import numpy as np
+import cv2
+import face_alignment
+from torchvision.utils import save_image
+
+class KeypointsGenerator():
+    def __init__(self):        
+        self.to_tensor = transforms.ToTensor()
+        self.fa = face_alignment.FaceAlignment(face_alignment.LandmarksType._2D, flip_input=True, device='cpu')
+
+    def get_keypoints(self, input_frames, image_size = 256, crop_data = False):
+        """Generates dataset images, keypoints (also called poses), and segmenatations from input_frames/ frames
+
+        Inputs
+        ----------
+        input_frames: list of images
+        crop_data : A flag used center-crop output images and poses (the original paper used crop_data=True,
+                    so for consistency we use crop_data=True as well) 
+
+        Returns
+        -------
+        poses:  torch tensor of keypoints 
+
+        """        
+        poses = []
+        # Finding the batch-size of the input imgs
+        if len(input_frames.shape) == 3:
+            input_frames = input_frames[None]
+            N = 1
+
+        else:
+            N = input_frames.shape[0]
+
+        # Iterate over all the images in the batch
+        for i in range(N):
+            
```

Comment:
No line here please. At this point, I think you should be able to look out for these yourself. Please don't wait until I need to point it out. Any code in this codebase should strictly conform to their code standards.